util/auto_roll_logger_test.cc

```diff
@@ -6,7 +6,7 @@
 
 #ifndef ROCKSDB_LITE
 
-#include "db/auto_roll_logger.h"
+#include "util/auto_roll_logger.h"
```

Comment:
can we remove `#include "db/db_test_util.h"` on line 20? I think it's compilable without it.